compiler/il/OMRNode.hpp

```diff
@@ -373,6 +364,22 @@ class OMR_EXTENSIBLE Node
       TR_ASSERT(opvalue != TR::vconst, "use createVectorConst to create node: %s", opcode.getName());
       return true;
       }
+      
+   static bool isNotDeprecatedUnsigned(TR::ILOpCodes opvalue)
+      {
+      
+      TR::ILOpCodes DeprecatedUnsignedList [] = 
```

Comment:
per other PRs in your series please convert to a switch statement on the opcode.

Reply:
Changed the structure of this function in https://github.com/eclipse/omr/commit/1a4a32c95f46392f0c73f8f16865f7314df1a3c1. Keep the structure consistent with the same function in #4367. 
FYI, there are 2 other WIP PRs also introducing the same function using the old structure. I will unWIP those PRs and resolve the conflicts there to update this function once this PR or #4367 has been merged.